NodeManager.ino

```diff
@@ -35,6 +35,54 @@ void before() {
   */
   
 
+   nodeManager.setRebootPin(A0);
```

Comment:
Clever, are you proposing to dump some common directives/examples in the main sketch so the user can just comment it out? I like the idea